firefox-ios/firefox-ios-tests/Tests/XCUITests/MicrosurveyTests.swift

```diff
@@ -89,22 +89,6 @@ final class MicrosurveyTests: BaseTestCase {
     }
 
     private func generateTriggerForMicrosurvey() {
-        let homepageToggleButtonIphone =
-        app.buttons[AccessibilityIdentifiers.FirefoxHomepage.OtherButtons.privateModeToggleButton]
-        let homepageToggleButtonIpad = app.buttons[AccessibilityIdentifiers.Browser.TopTabs.privateModeButton]
-        if !iPad() {
-            homepageToggleButtonIphone.waitAndTap()
-            mozWaitForElementToExist(app.staticTexts[AccessibilityIdentifiers.PrivateMode.Homepage.link])
-        } else {
-            homepageToggleButtonIpad.waitAndTap()
-            mozWaitForElementToExist(app.collectionViews[AccessibilityIdentifiers.Browser.TopTabs.collectionView])
-        }
-        if !iPad() {
-            homepageToggleButtonIphone.waitAndTap()
-            mozWaitForElementToExist(app.collectionViews[AccessibilityIdentifiers.FirefoxHomepage.collectionView])
-        } else {
-            homepageToggleButtonIpad.waitAndTap()
-            mozWaitForElementToExist(app.collectionViews[AccessibilityIdentifiers.Browser.TopTabs.collectionView])
-        }
+        navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
```

Comment:
this is temporary and not the right fix, @clarmso if there's a quick way to mimic the toggle, feel free to comment here. Otherwise I'll investigate tomorrow morning!